tests/model_hub_tests/transformation_tests/test_pa_transformation.py

```diff
@@ -45,7 +59,7 @@ def run_pa(tmp_path, model_id, model_link, use_block_indices_inputs, use_score_o
                 if re.search(block_indices_pattern, name):
                     block_indices_counter += 1
 
-        assert(block_indices_counter == pa_counter)
+        assert(block_indices_counter == resulting_map["PagedAttentionExtension"])
```

Comment:
print more verbose messages

The number of BlockIndices is expected to be equal to the number of PagedAttentionExtension ops 
Block indices cnt: block_indices_counter
 Paged Attention cnt: resulting_map["PagedAttentionExtension"]

Reply:
Yeah, I need  to check in this place and add if needed

Reply:
Added a more verbose assertion message:

![image](https://github.com/user-attachments/assets/2c71ca75-5d09-44a2-9e1a-b794ff7fde69)